duckduckgo/danger-settings
# Update node modules to make local setup work

Task/Issue URL: https://app.asana.com/0/1203301625297703/1204433323630035/f
Tech Design URL:
CC:

Description:
Update Node setup to make Danger work locally.

How to test: 
- Checkout this branch locally.
- Run `yarn install` to setup your local environment 
- Run `yarn jest` and verify it’s green. It runs the test suite.
- Find an open PR the iOS repository and run `yarn danger pr https://github.com/duckduckgo/iOS/pull/<PR NUMBER> --dangerfile ./org/allPRs.ts` and verify Danger runs locally without errors. 

- [This](https://github.com/duckduckgo/iOS/pull/1660) test PR in the iOS repo uses the Danger setup from this branch. Verify that it’s all green.
